pkg/webhook/configuration.go

```diff
@@ -74,6 +74,16 @@ var (
 	transformConfig = []transformConfigFunc{
 		addCSPCDeleteRule,
 	}
```

Comment:
add `addCVCWithUpdateRule` to transformConfig.

Reply:
done